apihub/provision-apihub.sh

```diff
@@ -22,7 +22,18 @@
 # For more about the Registry Provisioning API that is used in this
 # script, see https://cloud.google.com/apigee/docs/api-hub/provision.
 
-# Usage: provision-apihub.sh -p GCP-PROJECT-ID -l us-central1 -u GCP-USER-EMAIL -r KEY-RING-NAME -k KEY-NAME
+# Prerequisites:
+# - A Google Cloud account (USER_EMAIL) with billing enabled.
+# - A Google Cloud project (PROJECT_ID).
+# - The gcloud tool (https://cloud.google.com/sdk/gcloud).
+# - A chosen location (RUNTIME_LOCATION) for the instance. Currently "us-central1"
+#   is recommended. Other supported locations are listed at
+#     https://cloud.google.com/apigee/docs/api-hub/provision
+# - A chosen CMEK (https://cloud.google.com/kms/docs/cmek) key ring name
```

Comment:
I would add that the CMEK needs to be in the same region as the RUNTIME_LOCATION

Reply:
I realized that this script is always creating the CMEK (in the location chosen by the customer) and doesn't support the use case where customer already has an encryption key that they would like to reuse. In the latter case, what Gitesh mentioned above would be a requirement.

We can also address this in a follow up CL.

Reply:
sounds good to me. As long as the CMEK is all created by the script in the correct region I am good with this CL.